src/deep-cache/lib/Driver/S3FSDriver.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Created by CCristi <ccovali@mitocgroup.com> on 12/10/15.
+ */
+
+'use strict';
+
+import {AbstractFsDriver} from './AbstractFsDriver';
+import {Exception} from '../Exception/Exception';
+import path from 'path';
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['path' is defined but never used.](https://www.codacy.com/app/MitocGroup/deep-framework/file/1585049085/issues/source?bid=2895475&fileBranchId=3024729#l9)